packages/table/src/react/__specs__/__snapshots__/storyshots.spec.js.snap

```diff
@@ -867,8 +867,7 @@ exports[`Storyshots columnHeader sort asc w/ onClick 1`] = `
   >
     <div
       aria-sort="ascending"
-      data-css-1yecw7w=""
-      onClick={[Function]}
```

Comment:
This was the cause of the bug we had click handler being passed by filterReactProps to the wrapping div and and the same one on the button.